src/accelerate/components/Link.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+
+type Props = {
+	url: string,
+	location: string,
+	linkText: string,
+};
+
+export default function Link( props: Props ) {
+	return (
+		<a
+			href={ props.url }
+			onClick={ () => {
+				analytics.track(
+					'click',
+					{
+						location: props.location,
+						url: props.url,
+						link_text: props.linkText,
+					}
+				);
+			}}
+		>
+			{ __( props.linkText, 'altis-analytics' ) }
```

Comment:
Variables can't be used in translation functions - they are analysed / extracted from code using static analysis.